File: src/main/kotlin/se/teamkugel/julhaxx/IUserDetailsRepo.java
package se.teamkugel.julhaxx;

import se.teamkugel.julhaxx.MattesUserDetailsManager.MutableUser;

public interface IUserDetailsRepo {
	MutableUser get(String key);
	void put(String key, MutableUser mutableUser);
	void remove(String key);
	boolean containsKey(String key);
}

File: src/main/kotlin/se/teamkugel/julhaxx/IUserDetailsRepo.kt
package se.teamkugel.julhaxx

interface IUserDetailsRepo {
    operator fun get(key: String): MattesUserDetailsManager.MutableUser
    fun put(key: String, mutableUser: MattesUserDetailsManager.MutableUser)
    fun remove(key: String)
    fun containsKey(key: String): Boolean
}